### DISCUSSION TITLE
How to run CB-Operator on k8s environment?

### DISCUSSION THREAD

**Discussion by inno-cloudbarista** [cloud-barista/cb-operator#155] [Q&A]
CB-Operator를 k8s 환경에서 구동하기 위해 가이드(https://github.com/cloud-barista/cb-operator/wiki/features-and-usages) 를 따라 operator 바이너리를 빌드 후 ./operator pull 명령어를 수행하였습니다. 하지만 
![image](https://user-images.githubusercontent.com/69338107/132635507-4e8aea91-0797-4525-b4d0-2397a62472dc.png)
그림과 같은 에러가 발생하고 있습니다. 해당 문제를 해결하기 위해서 추가적인 설정이 필요한가요~?

**Comment by jihoon-seo** [comment]
@inno-cloudbarista 이슈 리포트 감사합니다.
일단은 pull 없이 바로 run 하시면,
자동으로 필요한 이미지를 pull 하고 run 할 것으로 예상됩니다. 😊

**Reply by jihoon-seo** [reply]
Fixed by the PR #156
